src/locales/pt.js

```diff
@@ -0,0 +1,706 @@
+    'widget.help': 'A seção de WIDGET permite você gerenciar um widget existente ou adicionar um novo widget. Widgets são objetos usados para mostrar informação e funcionalidade do sistema ou representam serviços de uma página da aplicação.',
+    'fragment.list.title': 'Fragmentos',
+    'fragment.help': 'A seção FRAGMENTOS permite você gerenciar um fragmento existente ou adionar novos fragmentos. Os fragmentos são um porção simples da interface de front-end usado nos Widgets (incluindo Plugins) e nos Modelos de Página.',
+    'fragment.code': 'Código',
+    'fragment.code.placeholder': 'Código',
+    'fragment.tab.guiCode': 'Código de Interface',
+    'fragment.tab.defaultGuiCode': 'Interface Padrão',
+    'fragment.body.defaultGuiCode': 'Indisponível.',
+    'fragment.detail.title': 'Detalhes do fragmento',
+    'fragment.detail.widgetType': 'Tipo do Widget',
+    'fragment.detail.pluginCode': 'Código do Plugin',
+    'fragment.detail.title.referencedFragments': 'Fragmento referenciados',
+    'fragment.detail.title.referencedPageModels': 'Modelos de Página referenciados',
+    'fragment.detail.title.referencedWidgetType': 'Tipos de Widget referenciados',
+    'fragment.detail.emptyReferenceFragments': 'Não há fragmentos referenciados',
+    'fragment.detail.emptyReferencePageModels': 'Não há modelos de página referenciados',
+    'fragment.detail.emptyReferenceWidgetTypes': 'Não há tipos de widget referenciados',
+    'fragment.form.edit.plugin': 'Plugin',
+    'fragment.table.widgetType': 'Tipo de Widget',
+    'fragment.table.plugin': 'Plugin',
+    'fragment.table.edit': 'Editar {code}',
+    'fragment.table.details': 'Detalhes de: {code}',
+    'fragment.form.edit.widgetType': 'Tipo de Widget',
+    'fragment.settings': 'Habilitar Edição de Fragmento com interface padrão em branco',
+    'fragment.settings.alert.success': 'As configurações foram atualizadas com sucesso',
+    'fragment.settings.alert.error': 'As configurações não foram atualizadas',
+    'fragment.alert.error.fileExtension': 'Essa extenção não é permitida',
+    'fileBrowser.list.upLink': 'acima..',
+    'fileBrowser.list.size': 'Tamanho',
+    'fileBrowser.list.lastModifiedTime': 'Ultima modificação',
+    'fileBrowser.list.empty': 'Está pasta está vazia',
+    'fileBrowser.createTextFile': 'Criar arquivo de texto',
+    'fileBrowser.createFolder': 'Criar pasta',
+    'fileBrowser.uploadFile': 'Upload',
+    'fileBrowser.uploadFileComplete': 'Upload de arquivo concluído',
+    'fileBrowser.uploadFileError': 'Erro no upload de arquivo',
+    'fileBrowser.downloadFile': 'Download',
+    'fileBrowser.newFolder': 'Nova da Nova Pasta',
+    'fileBrowser.createFolderSuccess': 'Pasta \'{path}\' criada com sucesso',
+    'fileBrowser.createFolderError': 'Ocorreu um erro ao criar a pasta \'{path}\'',
+    'fileBrowser.deleteFolderSuccess': 'Pasta \'{path}\' deletada com sucesso',
+    'fileBrowser.deleteFolderError': 'Ocorreu um erro ao deletar a pasta \'{path}\'',
```

Comment:
I think we should use the term `remover` instead of `deletar`.

Reply:
Totally agreed